packages/core/src/input/pointer/emitter/PointerUIEventEmitter.ts

```
@@ -0,0 +1,237 @@
+          if (canvas.raycast(ray, hitResult, farClipPlane)) {
+            this._updateRaycast(<Component>hitResult.component, pointer);
+            return;
+          }
+        }
+        if (camera.clearFlags & CameraClearFlags.Color) {
+          this._updateRaycast(null);
+          return;
+        }
+      }
+    }
+    return null;
```

Biome

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)